src/style/ParagraphProperties.ts

```diff
@@ -127,6 +139,20 @@ export class ParagraphProperties implements IParagraphProperties {
     }
   }
 
+  /**
+   * Sets the keep together attribute if paragraph marked as keep together.
+   *
+   * @param {Element} textPropertiesElement The element which will take the attribute
+   */
+  private setKeepTogetherAttribute(paragraphPropertiesElement: Element): void {
+    if (this.shouldKeepParagraphTogether === true) {
+      paragraphPropertiesElement.setAttribute(
+        OdfAttributeName.FormatKeepTogether,
+        "always"
+      );
```

Comment:
max line length is 120, so you can put the whole in one line

Reply:
Yeah, that's my local eslint kicking in.

Reply:
Addressed in 8aa67a03bd379c1754f98c13d136d76d23bd89bf